sidebars.json

```diff
@@ -916,6 +916,42 @@
           "id": "getting-started/tutorials/project-template-integrated-with-jenkins",
           "type": "doc"
         },
+        {
+          "label": "AuthN/AuthZ Architecture",
+          "type": "category",
+          "collapsed": true,
+          "link": {
+            "type": "generated-index",
+            "title": "AuthN/AuthZ Architecture ",
+            "slug": "/getting-started/tutorials/auth-architecture"
+          },
+          "items": [
+            {
+              "id": "getting-started/tutorials/auth-architecture/intro",
+              "type": "doc"
+            },
+            {
+              "id": "getting-started/tutorials/auth-architecture/static-api-key",
+              "type": "doc"
+            },
+            {
+              "id": "getting-started/tutorials/auth-architecture/external-idp",
+              "type": "doc"
+            },
+            {
+              "id": "getting-started/tutorials/auth-architecture/external-idp-internal-session",
+              "type": "doc"
+            },
+            {
+              "id": "getting-started/tutorials/auth-architecture/multiple-projects",
+              "type": "doc"
+            },
+            {
+              "id": "getting-started/tutorials/auth-architecture/enterprise-architecture",
```

Comment:
I'd change the path to prevent having the `auth` contraction and `architecture` duplicated.

Also if we plan to have other tutorials for the architecture topic I'd consider putting that first in the URL @akelity what do you think about it?

What about: `getting-started/tutorials/architecture/auth/[enterprise|multiple-projects|...]`


Also I'm not sure about using the `auth` contraction, as far as I understood is to be inclusive of both authentication and authorization, isn't it? 🤔

Reply:
I agree. I have changed the path as suggested